nsqd/http.go

```diff
@@ -596,3 +600,41 @@ func (s *httpServer) printStats(stats []TopicStats, health string, startTime tim
 	}
 	return buf.Bytes()
 }
+
+type allowedOpts struct {
+	NSQLookupdTCPAddresses []string `json:"nsqlookupd_tcp_addresses"`
+	Verbose                bool     `json:"verbose"`
```

Comment:
It'd be cool if this were a pointer, so I don't accidentally set Verbose false by default when updating the lookupd addresses, especially if I want it to be true.